releng/release-notes-next/chroot_scan_tarball.feature

```diff
@@ -0,0 +1,3 @@
+New ``tarball`` option for ``chroot_scan`` plugin. Without it, directory
```

Comment:
This si markdown, not sure if double-quoting works, single quotes work for sure.